__tests__/unit/core-transaction-pool/connection.test.ts

```diff
@@ -217,8 +215,8 @@ describe("Connection", () => {
         it("should add the transactions to the pool and they should expire", async () => {
             expect(connection.getPoolSize()).toBe(0);
 
-            const expireAfterSeconds = 3;
-            const expiration = Slots.getTime() + expireAfterSeconds;
+            const expireAfterBlocks: number = 3;
+            const expiration: number = /* XXX how to get current height? */ 123 + expireAfterBlocks;
```

Comment:
This is a unit test so you can just use a fixed value, no need to get a real height.